src/three-components/PMREMGenerator.ts

```diff
@@ -41,61 +45,81 @@ export const generatePMREM =
             target.dispose();
           });
           meshes.forEach((mesh) => {
-            (mesh.material as Material).dispose();
+            // (mesh.material as Material).dispose();
```

Comment:
Did we decide that we aren't going to dispose of this stuff? If so, maybe the meshes should just be a static array that gets updated as needed?

Reply:
The shaders take longer to compile than they take to run, so it's very important that we don't dispose of them, since three does such a nice job caching them. Are we worried about a leak here? The programs are always identical, so three ensures they don't get duplicated. We shouldn't delete them until <model-viewer> will not be loading any new environments anymore, and basing that on page navigation sounds reasonable?